### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by Boxface**

### CODE REVIEW

File: src/components/Modal/Modal.styles.js [hunk 1]
```diff
@@ -1,11 +1,63 @@
...
+	}}
 	border-radius: 10px;
+	color: var(--color-white);
 	left: 50%;
 	padding: var(--spacing-large);
 	position: fixed;
 	top: 50%;
 	transform: translate(-50%, -50%);
+	z-index: 3;
+	input {
```
Comment (dan1elhughes) [inline src/components/Modal/Modal.styles.js]: These nested selectors should be split into styled components

File: src/components/SettingsPanel/assets/SettingPanel-Chevron-Black.svg [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="444.819px" height="444.819px" viewBox="0 0 444.819 444.819" style="enable-background:new 0 0 444.819 444.819;"
+	 xml:space="preserve">
+<g>
+	<path d="M352.025,196.712L165.884,10.848C159.029,3.615,150.469,0,140.187,0c-10.282,0-18.842,3.619-25.697,10.848L92.792,32.264
+		c-7.044,7.043-10.566,15.604-10.566,25.692c0,9.897,3.521,18.56,10.566,25.981l138.753,138.473L92.786,361.168
+		c-7.042,7.043-10.564,15.604-10.564,25.693c0,9.896,3.521,18.562,10.564,25.98l21.7,21.413
+		c7.043,7.043,15.612,10.564,25.697,10.564c10.089,0,18.656-3.521,25.697-10.564l186.145-185.864
+		c7.046-7.423,10.571-16.084,10.571-25.981C362.597,212.321,359.071,203.755,352.025,196.712z"/>
+</g>
```
Comment (dan1elhughes) [inline src/components/SettingsPanel/assets/SettingPanel-Chevron-Black.svg]: Can we get rid of all these empty groups

File: src/tokens.yaml [hunk 3]
```diff
@@ -32,8 +34,12 @@ radius:
     medium: 10px
     small: 5px
 shadow:
-    dark: "rgba(0, 0, 0, 0)"
-    light: "#f0f3ff"
+    tile:
+        dark: "rgba(255,255,255,.14)"
+        light: "rgba(0,0,0,.14)"
```
Comment (dan1elhughes) [inline src/tokens.yaml:39]: A lot of duplicates here, can they be pulled out into their own token

**Review by dan1elhughes — CHANGES_REQUESTED** [review]

File: src/components/Modal/Modal.styles.js (hunk 1 above)
Comment (Boxface) [reply src/components/Modal/Modal.styles.js]: All nested selectors from this styles.js file are now styled components in the SettingsPanel.styles.js

File: src/components/SettingsPanel/assets/SettingPanel-Chevron-Black.svg (hunk 2 above)
Comment (Boxface) [reply src/components/SettingsPanel/assets/SettingPanel-Chevron-Black.svg]: Removed unneeded SVGs

File: src/tokens.yaml (hunk 3 above)
Comment (Boxface) [reply src/tokens.yaml:39]: Removed unneeded shadow tokens

**Review by dan1elhughes — APPROVED** [review]

**Comment by dan1elhughes** [comment]
Fixes #12

**Pull request merged**